qualtrix/client.py

```diff
@@ -232,6 +232,67 @@ def get_response(survey_id: str, response_id: str):
     return survey_answers
 
 
+def get_contact(contact_id: str):
+    r = requests.get(
+        settings.BASE_URL
+        + f"/directories/{settings.DIRECTORY_ID}/contacts/{contact_id}",
+        headers=auth_header,
+        timeout=settings.TIMEOUT,
+    )
+
+    return r.json()
+
+
+def get_contact_history(contact_id: str):
+
+    logging.info(f"contact history for {contact_id}")
+    r = requests.get(
+        settings.BASE_URL
+        + f"/directories/{settings.DIRECTORY_ID}/contacts/{contact_id}/history",
+        headers=auth_header,
+        timeout=settings.TIMEOUT,
+        params={"type": "response"},
+    )
+
+    logging.info(f"contact history for {contact_id}")
```

Comment:
redundant, and only gives information this endpoint was hit. Maybe log some information about the response from qualtrics?